Update readme (#8)
kyma-incubator/terraform-provider-gardener: examples/gcp/README.md

> # Gardener GCP Example

Reviewer: ```suggestion
# Gardener GCP
```

Author: applied change

> This is a configuration example to deploy an gcp cluster using gardener. You need to make sure that the secret binding parameter is "gcp_secret_binding" for the deployment to work properly and that the resource is "gardener_gcp_shoot".

Reviewer: ```suggestion
Use this configuration example to deploy a GCP cluster using Gardener.
 >**NOTE**: For the example to work properly, make sure that the **secret_binding** parameter is set to `gcp_secret_binding` and that the **resource** is `gardener_gcp_shoot`.
```

Author: applied change

> For now the following cidrs are obtrained from the similar clusters deployed on gardener, in the future these values should be obtained programatically.

Reviewer: ```suggestion
Currently, values for the following CIDRs are obtained from the similar clusters deployed on Gardener:
```

```bash
   workerscidr       = ["10.250.0.0/19"]
```

> multiple workers can be defined as follows

Reviewer: ```suggestion
You can also define multiple workers:
```

Author: applied change
